functions/src/utils/firebase/firebase.ts

```diff
@@ -0,0 +1,133 @@
+import * as firebase from 'firebase';
+import 'firebase/auth';
+import 'firebase/firestore';
+
+interface Document {
+    id: string
+    data: () => {}
+}
+
+interface Filter {
+    key: string
+    match: string
+    value: any
+}
+
+const firebaseConfig = {
+    apiKey: "AIzaSyA_AQVxsSjqXgsVQgUIIOyNGvXpXY-9FtY",
+    authDomain: "main-street-project-qa.firebaseapp.com",
+    databaseURL: "https://main-street-project-qa.firebaseio.com",
+    projectId: "main-street-project-qa",
+    storageBucket: "main-street-project-qa.appspot.com",
+    messagingSenderId: "941395738080",
+    appId: "1:941395738080:web:694c42266e474dd2553f85",
+    measurementId: "G-F0J5X7MPXP"
```

Comment:
I've never worked with Firebase before, but it's generally a bad practice to commit any kind of secrets directly to source control

Reply:
I'll change the API key and deliver it via remote config before going to prod